src/MvoContaoFacebookImportBundle.php

```diff
@@ -21,4 +21,8 @@
  */
 class MvoContaoFacebookImportBundle extends Bundle
 {
+    public function getPath(): string
+    {
+        return \dirname(__DIR__);
+    }
```

Comment:
This change seems unrelated.

Reply:
see: https://github.com/m-vo/contao-facebook-import/pull/33